### ISSUE TITLE
lando start/rebuild:  [ErrorException]   file_put_contents

### ISSUE THREAD

**Issue by ptmkenny** [lando/lando#348]
Bug Report
==========

I am getting a ` [ErrorException]
  file_put_contents(/app/vendor/bin/php-parse): failed to open stream: No such file or directory` when attempting to start/rebuild my lando app.

Windows 10 Pro, running lando beta17 from powershell.

The error occurs when I run `lando rebuild`. I also tried running `lando destroy`, `lando poweroff`, and then `lando start`; this resulted in the same error.

**lando.yml:**

```
name: mysite
recipe: pantheon
config:
  framework: drupal8
  xdebug: true
  env: dev
  site: mysite
  id: myid
  webroot: web/

proxy:
  mailhog:
    - mail.mysite.lndo.site
services:
  mailhog:
    type: mailhog
    hogfrom:
      - appserver
    portforward: 1026

  appserver:
    extras:
      - "apt-get update -y"
      - "apt-get install vim -y"
    overrides:
      services:
        environment:
          BEHAT_PARAMS: '{"extensions" : {"Behat\\MinkExtension" : {"base_url" : "http://edosensei.lndo.site/"}, "Drupal\\DrupalExtension" : {"drush" :   {  "root":  "/app/web" }}}}'

tooling:
  behat:
    service: appserver
    description: Run behat tests locally.
    cmd:
      - /app/vendor/bin/behat
```

Here is the error:

```
Gathering patches for root package.
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating optimized autoload files
Warning: Ambiguous class resolution, "Drupal\Component\Utility\Random" was found in both "$baseDir . '/web/core/lib/Drupal/Component/Utility/Random.php" and "/app/vendor/drupal/drupal-driver/src/Drupal/Component/Utility/Random.php", the first will be used.
Removing packages services cache file:
/app/vendor/drupal/console/extend.console.uninstall.services.yml
Creating packages services cache file:
/app/vendor/drupal/console/extend.console.uninstall.services.yml
> DrupalComposer\DrupalScaffold\Plugin::scaffold
Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)Downloading (100%)> DrupalProject\composer\ScriptHandler::createRequiredFiles


  [ErrorException]
  file_put_contents(/app/vendor/bin/php-parse): failed to open stream: No such file or directory


install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

error:  Error: 1
    at ChildProcess.<anonymous> (C:\snapshot\cli\lib\shell.js:143:16)
    at emitTwo (events.js:125:13)
    at ChildProcess.emit (events.js:213:7)
    at maybeClose (internal/child_process.js:887:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:208:5)
From previous event:
    at spawn (C:\snapshot\cli\lib\shell.js:93:10)
    at Object.exports.sh (C:\snapshot\cli\lib\shell.js:209:12)
    at shCompose (C:\snapshot\cli\lib\compose.js:24:16)
    at Object.exports.run (C:\snapshot\cli\lib\compose.js:343:10)
    at C:\snapshot\cli\lib\docker.js:163:22
    at runCallback (timers.js:800:20)
    at tryOnImmediate (timers.js:762:5)
    at processImmediate [as _immediateCallback] (timers.js:733:5)
From previous event:
    at Object.exports.run (C:\snapshot\cli\lib\docker.js:150:18)
    at C:\snapshot\cli\lib\engine.js:526:19
    at runCallback (timers.js:800:20)
From previous event:
    at Object.exports.run (C:\snapshot\cli\lib\engine.js:525:4)
    at AsyncEvents.<anonymous> (C:\snapshot\cli\plugins\lando-services\lib\config.js:148:31)
    at AsyncEvents.handle (C:\snapshot\cli\lib\events.js:109:19)
    at C:\snapshot\cli\lib\events.js:170:19
From previous event:
    at AsyncEvents.emit (C:\snapshot\cli\lib\events.js:161:18)
    at C:\snapshot\cli\lib\app.js:767:23
From previous event:
    at Object.exports.start (C:\snapshot\cli\lib\app.js:766:4)
    at C:\snapshot\cli\lib\app.js:1047:17
    at runCallback (timers.js:800:20)
    at tryOnImmediate (timers.js:762:5)
    at processImmediate [as _immediateCallback] (timers.js:733:5)
From previous event:
    at Object.exports.rebuild (C:\snapshot\cli\lib\app.js:1046:4)
    at C:\snapshot\cli\plugins\lando-core\lib\tasks\rebuild.js:56:28
From previous event:
    at Object.run (C:\snapshot\cli\plugins\lando-core\lib\tasks\rebuild.js:48:8)
    at C:\snapshot\cli\lib\tasks.js:173:21
From previous event:
    at C:\snapshot\cli\lib\tasks.js:172:8
    at runCallback (timers.js:800:20)
    at tryOnImmediate (timers.js:762:5)
    at processImmediate [as _immediateCallback] (timers.js:733:5)
From previous event:
    at Object.exports.parseToYargs.command.handler (C:\snapshot\cli\lib\tasks.js:157:6)
    at Object.module.exports.self.runCommand (C:\snapshot\cli\node_modules\yargs\lib\command.js:170:22)
    at parseArgs (C:\snapshot\cli\node_modules\yargs\yargs.js:920:28)
    at Object.get [as argv] (C:\snapshot\cli\node_modules\yargs\yargs.js:860:16)
    at C:\snapshot\cli\lib\cli.js:271:7
From previous event:
    at Object.exports.init (C:\snapshot\cli\lib\cli.js:238:4)
    at C:\snapshot\cli\bin\lando.js:21:20
From previous event:
    at Object.<anonymous> (C:\snapshot\cli\bin\lando.js:20:2)
    at Module._compile (pkg/prelude/bootstrap.js:1226:22)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1281:12)
    at startup (bootstrap_node.js:200:16)
    at bootstrap_node.js:617:3
```

I will continue to try to debug this.  I tried rolling my repo back to a commit yesterday (when I wasn't having trouble with lando) but when I rebuild, I'm still getting the same error.

EDIT: I used `lando destroy`, uninstalled lando, rolled back to beta-13, rebooted the machine, and I am still getting the same error when I run `lando rebuild`. I reset docker to factory defaults (and then enabled shared drives again) but that didn't help, either.  I noticed Windows 10 had applied a couple updates, so I rolled those back, but still no change.

So, I'm thinking that there is some lingering problem in my configuration that did not get cleared by the various resets I have tried.

Also, after this error, I can start the lando app with `lando start`, but some things are broken, For example, if I ssh into the appserver with `lando ssh appserver` and go to `/app/vendor/bin`, the symbolic links are broken:

```
www-data@a2ac67b19124:/app/vendor/bin$ ls -la
ls: cannot read symbolic link behat: No such file or directory
ls: cannot read symbolic link drupal: No such file or directory
ls: cannot read symbolic link drush: No such file or directory
ls: cannot read symbolic link drush.complete.sh: No such file or directory
ls: cannot read symbolic link drush.launcher: No such file or directory
ls: cannot read symbolic link drush.php: No such file or directory
ls: cannot read symbolic link php-parse: No such file or directory
ls: cannot read symbolic link phpunit: No such file or directory
ls: cannot read symbolic link psysh: No such file or directory
```

**Comment by ptmkenny** [comment]
**TL;DR: Lando/docker uninstall doesn't fix this.  Fixed by deleting the repo and cloning from github again.**

Results of more testing:

Ran `lando destroy`, uninstalled Lando, Docker, and Git for Windows.  Reinstalled lando and ran `lando start`; had the same problem.

Installed lando from scratch on a different Win10 machine, ran `lando start`, everything was fine.

Went back to the original machine, deleted the repo, re-cloned from github... and it's fine now.  But, I have no idea why, because the original repo was (in theory) in the same state, because I had reset it with `git reset HEAD --hard`.
